docs/current/cli/465058-install.md

```diff
@@ -0,0 +1,129 @@
+---
+slug: /cli/465058/install
+---
+
+# Installation
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import CodeBlock from "@theme/CodeBlock";
+import styles from "@site/src/css/install.module.scss";
+
+:::tip
+The `dagger` CLI is available for installation on macOS, Linux, and Windows to run locally or in a CI environment.
+:::
+
+Install the `dagger` CLI following the steps below.
+
+<BrowserOnly>
+{() =>
+<Tabs defaultValue={
+ window.navigator.userAgent.indexOf('Linux') != -1 ? 'linux':
+ window.navigator.userAgent.indexOf('Win') != -1 ? 'windows':
+ 'macos'}
+groupId="os"
+values={[
+{label: 'macOS', value: 'macos'}, {label: 'Linux', value: 'linux'}, {label: 'Windows', value: 'windows'},
+]}>
+
+<TabItem value="macos">
+
+We assume that you have [Homebrew](https://brew.sh/) installed.
+If you do, you can install `dagger` with a single command:
+
+```shell
+brew install dagger
```

Comment:
We should encourage users to use our tap instead of installing from the bottle formuale. Check instructions here https://docs.dagger.io/1001/install/

Reply:
Done